src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -35,4 +35,10 @@ export const esmPatientRegistrationSchema = {
       },
     ],
   },
+  registrationSubmission: {
+    afterCreateUrl: {
```

Comment:
```suggestion
  links: {
    submitButton: {
```

Remember that the config is an interface for non-developers. It should be easy to understand just by looking at the UI, without knowing anything about the code.

Reply:
Makes sense to me.

Reply:
Fixed